Description: Fix CMake-related build problem
 With the old CMP0054 behavior "t" in cmake/CMakeCSharpInformation.cmake:311
 is expanded to the value of the t variable and the elseif clause behaves
 incorrectly.
Author: Andrey Rahmatullin <wrar@debian.org>
Bug-Debian: https://bugs.debian.org/844956
Last-Update: 2016-11-20

--- libproxy-0.4.13.orig/CMakeLists.txt
+++ libproxy-0.4.13/CMakeLists.txt
@@ -3,6 +3,9 @@ cmake_minimum_required(VERSION 2.6)
 if(POLICY CMP0011)
    cmake_policy(SET CMP0011 NEW)
 endif(POLICY CMP0011)
+if(POLICY CMP0054)
+   cmake_policy(SET CMP0054 NEW)
+endif(POLICY CMP0054)
 
 # Make sure we look in our cmake folder for additional definitions
 set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} ${CMAKE_SOURCE_DIR}/cmake )
